runtime/makelib/targets.mk.aix.inc.ftl

```diff
@@ -98,13 +102,20 @@ else
 endif
 
 ifeq (,$(findstring xlclang,$(notdir $(CC))))
-  # xlc options
-  CFLAGS += -q mbcs -qinfo=pro
+  ifneq ($(OMR_ENV_OPENXL),1)
+    # xlc options
+    CFLAGS += -q mbcs -qinfo=pro
+  endif
 else
- # xlclang options
+  # xlclang options
   CFLAGS += -qxlcompatmacros
 endif
-CFLAGS += -qlanglvl=extended -qarch=ppc -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108 -qstackprotect
+ifneq ($(OMR_ENV_OPENXL),1)
+  CFLAGS += -qlanglvl=extended -qarch=ppc -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108 -qstackprotect
+else
+  #openxl options
+  CFLAGS += -std=c89 -qarch=ppc -fno-strict-aliasing -fstack-protector
+endif
```

Comment:
Please invert this to avoid the negation:
```
ifeq ($(OMR_ENV_OPENXL),1)
  # openxl options
  CFLAGS += -std=c89 -qarch=ppc -fno-strict-aliasing -fstack-protector
else
  CFLAGS += -qlanglvl=extended -qarch=ppc -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108 -qstackprotect
endif
```
Same comment near lines 128 & 139.